packages/utilities/fast-gestures/docs/api-report.md

```diff
@@ -0,0 +1,49 @@
+## API Report File for "@microsoft/fast-gestures"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export interface IPointer {
+    // (undocumented)
+    pointerId: number;
+    // Warning: (ae-forgotten-export) The symbol "IPointerInfo" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    pointerInfo: IPointerInfo;
+}
+
+// Warning: (ae-forgotten-export) The symbol "PointerGestureFeature" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type IPointerGestureConfig = Partial<Record<PointerGestureFeature, boolean>>;
+
+// Warning: (ae-forgotten-export) The symbol "IPointerGestureInfo" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type PointerGestureCallback = (pointerGestureInfo: IPointerGestureInfo) => void | null;
+
+// Warning: (ae-forgotten-export) The symbol "Disposable" needs to be exported by the entry point index.d.ts
```

Comment:
Can you make sure these ae-forgotten-export subjects are exported?

Reply:
Sure. I've addressed all the `ae-forgotten-export` warnings. Thanks!